pyrefly/lib/lsp/server.rs

```diff
@@ -221,16 +221,16 @@ impl ServerConnection {
     }
 }
 
-struct Server {
+pub struct Server {
     connection: ServerConnection,
     /// A thread pool of size one for heavy read operations on the State
     async_state_read_threads: ThreadPool,
     lsp_queue: LspQueue,
     initialize_params: InitializeParams,
     indexing_mode: IndexingMode,
     workspace_indexing_limit: usize,
-    state: Arc<State>,
-    open_files: Arc<RwLock<HashMap<PathBuf, Arc<String>>>>,
+    pub state: Arc<State>,
```

Comment:
Do you think it would be better to add functions to access the necessary information?